docs/Contributing/Blueprints/Fleet-Installers.md

```diff
@@ -0,0 +1,387 @@
+Though Zach has ran the following test with no issues:
+> I was able to Notarize ~200 packages in under 12 hours yesterday with no apparent limiting. 
+> Note this was using the older Notarization API (slower, about 3 mins per notarization).
+
+#### Future Optimization for Notarization
+
+This is the current structure of a `.pkg` generated with `fleetctl package` (unsigned and unnotarized):
+```sh
+pkg_expanded
+├── Distribution
+└── base
+    ├── Bom
+    ├── Library
+    │   └── LaunchDaemons
+    │       └── com.fleetdm.orbit.plist
+    ├── PackageInfo
+    ├── Payload
+    ├── Scripts
+    │   └── postinstall
+    └── opt
+        └── orbit
+            ├── bin
+            │   ├── desktop
+            │   │   └── macos
+            │   │       └── stable
+            │   │           ├── Fleet Desktop.app
+            │   │           │   └── Contents
+            │   │           │       ├── CodeResources
+            │   │           │       ├── Info.plist
+            │   │           │       ├── MacOS
+            │   │           │       │   └── fleet-desktop
+            │   │           │       └── _CodeSignature
+            │   │           │           └── CodeResources
+            │   │           └── desktop.app.tar.gz
+            │   ├── orbit
+            │   │   └── macos
+            │   │       └── stable
+            │   │           └── orbit
+            │   └── osqueryd
+            │       └── macos-app
+            │           └── stable
+            │               ├── osquery.app
+            │               │   └── Contents
+            │               │       ├── Info.plist
+            │               │       ├── MacOS
+            │               │       │   └── osqueryd
+            │               │       ├── PkgInfo
+            │               │       ├── Resources
+            │               │       │   └── osqueryctl
+            │               │       ├── _CodeSignature
+            │               │       │   └── CodeResources
+            │               │       └── embedded.provisionprofile
+            │               └── osqueryd.app.tar.gz
+            ├── certs.pem
+            ├── osquery.flags
+            ├── secret.txt
+            ├── staging
+            └── tuf-metadata.json
+```
+
+Worth mentioning that the only files that really change when users generate a pkg are:
+  - Library/LaunchDaemons/com.fleetdm.orbit.plist (contains the configuration, like URLs, etc.)
+  - opt/orbit/secret.txt (enroll secret)
+
+The `osqueryd.app.tar.gz` and `desktop.app.tar.gz`  files are used by Orbit to compare with remote targets when checking for updates.
+A future optimization could replace those `.app.tar.gz` files with a txt file with the hash of such file
+(would reduce ~30MB of uncompressed size reduction for the `.pkg`).
+
+### Windows
+
+Currently we don't support signing of the MSI installers in `fleetctl package`. But this functionality could be added both to the command and the service.
```

Comment:
When we decide to support this (which we should do soon), we can use https://github.com/mtrojnar/osslsigncode.